### ISSUE TITLE
ORTC invalid folder structure. ortc\xplatform\curl does not exist

### ISSUE THREAD

**Issue by djee-ms** [webrtc-uwp/webrtc-uwp-sdk#137]
When running `bin/prepare.bat` on Windows (branch `cse/m71-ks`) the build script immediately fails with the following error:
```
CRITICAL ERROR: ORTC invalid folder structure. ortc\xplatform\curl does not exist
```
Indeed the directory doesn't seem to exist anywhere, neither on disk nor by searching directly on GitHub the entire webrtc-uwp organization (all-repo search). All submodules are up to date and there is no local change.

However, on the same admin cmd.exe, opening `webrtc-uwp-sdk\webrtc\windows\solutions\WebRtc.Universal.sln` and compiling `WebRtc.UWP.Native.Builder` does not seem to care about that.

Am I missing something here?

**Comment by jamescadd** [comment]
Hi - prepare.bat is deprecated in this release and does not need to be run.  After a fresh clone you should be able to open the .sln in VS and build.

**Comment by djee-ms** [comment]
Yes I managed to build thanks. Shall we delete `prepare.bat` from the branch then? Or is it used for other things?
